DataSetExplorerTests/Unit/ImporterTests.cs

```diff
@@ -13,18 +16,50 @@ public class ImporterTests
         public void Imports_data_set_instances_and_annotations()
         {
             ExcelImporter importer = new ExcelImporter(new ExcelFactory().GetTestDataFolder());
-
             var dataSet = importer.Import("BurningKnight");
 
             var classes = dataSet.GetInstancesOfType(SnippetType.Class);
             classes.Count.ShouldBe(79);
             classes.Count(c => c.Annotations.Count == 3).ShouldBe(79);
-
+            
             var functions = dataSet.GetInstancesOfType(SnippetType.Function);
             functions.Count.ShouldBe(313);
             functions.Count(c => c.Annotations.Count == 1).ShouldBe(7);
             functions.Count(c => c.Annotations.Count == 2).ShouldBe(10);
             functions.Count(c => c.Annotations.Count == 3).ShouldBe(296);
+
+            var annotators = ImportAnnotators();
+            JoinInstancesAndAnnotators(ref classes, annotators);
+
+            var allClassAnnotations = classes.SelectMany(c => c.Annotations);
+            allClassAnnotations.First(a => a.Annotator.Id == 1).Annotator.ShouldBe(annotators.Find(a => a.Id == 1));
+            allClassAnnotations.First(a => a.Annotator.Id == 2).Annotator.ShouldBe(annotators.Find(a => a.Id == 2));
+            allClassAnnotations.First(a => a.Annotator.Id == 3).Annotator.ShouldBe(annotators.Find(a => a.Id == 3));
+        }
+
+        private void JoinInstancesAndAnnotators(ref List<DataSetInstance> annotatedInstances, List<Annotator> annotators)
+        {
+            foreach (var annotation in annotatedInstances.SelectMany(i => i.Annotations))
+            {
+                annotation.Annotator = annotators.Find(annotator => annotator.Id.Equals(annotation.Annotator.Id));
+            }
+        }
+
+        private List<Annotator> ImportAnnotators()
```

Comment:
Should this logic be in the test? Can we simply mock this data and put it in the _arrange_ section of the test instead of coupling it to this file?